### PR TITLE
Version 13 fixes v13-19-0

### PR DESCRIPTION
**Pull request by e-lobo**
Please Review and merge:
1. https://github.com/leam-tech/renovation_core/pull/81 ✅

Changes include but not limited to:
1. [Fix](https://github.com/leam-tech/frappe/pull/9/commits/f9c1872e7538397b578f7bdd015c68e1a84ba0b2) for is_owner 
2. [Fix](https://github.com/leam-tech/frappe/pull/9/commits/7ef73c324010fd1447d5b770a21f1fc824c1cb73) for test_runner when we run with flag --doctype
3. [Fix](https://github.com/leam-tech/frappe/pull/9/commits/a86e53f7e9360b7ae1698de7235b7f509b335d3a) validate_link for child tables
4. [Fix](https://github.com/leam-tech/frappe/pull/9/commits/830fc9c98a3acaaf665b380d7798f15196d6d865) import file ..
this is [fixed](https://github.com/frappe/frappe/commit/562a192e43a6a4d6fceb5a455ed76c796a96a55c)

⚠️ Warning: Please checkout pr mentioned above before updating if `renovation_core` is installed ⚠️

### CODE REVIEW

File: frappe/test_runner.py [hunk 1]
```diff
@@ -73,7 +74,9 @@ def main(app=None, module=None, doctype=None, verbose=False, tests=(),
 		else:
 			ret = run_all_tests(app, verbose, profile, ui_tests, failfast=failfast, junit_xml_output=junit_xml_output)
 
-		frappe.utils.scheduler.enable_scheduler()
+		if cint(frappe.db.get_value("System Settings", fieldname="enable_scheduler")):
+			frappe.utils.scheduler.disable_scheduler()
+			if frappe.db: frappe.db.commit()
```
Comment (assemmarwan) [inline frappe/test_runner.py]: @e-lobo Isn't this redundant?

Comment (e-lobo) [reply frappe/test_runner.py]: @assemmarwan this is to ensure any write operations in db are completed before tests start to run.. there is an issue if we remove frappe.db.commit() when we run tests with --doctype

Comment (e-lobo) [reply frappe/test_runner.py]: @assemmarwan fixed.. it was in the wrong place 👍

**Comment by e-lobo** [comment]
All good here @assemmarwan @fahimalizain @Abadulrehman

**Comment by Abadulrehman** [comment]
Looks good. 👍

**Review by Abadulrehman — APPROVED** [review]

**Review by assemmarwan — APPROVED** [review]

**Review by fahimalizain — APPROVED** [review]

**Pull request merged**